features/show_membership_application.feature

```diff
@@ -12,33 +12,47 @@ Feature: As an Admin
 
   Background:
     Given the following users exists
-      | email                       | admin |
-      | emma@random.com             |       |
-      | hans@random.com             |       |
-      | anna_needs_info@random.com  |       |
-      | lars_rejected@snarkybark.se |       |
-      | nils_member@bowwowwow.se    |       |
-      | admin@sgf.com               | true  |
+      | email                               | admin |
+      | emma@random.com                     |       |
+      | hans@random.com                     |       |
+      | anna_needs_info@random.com          |       |
+      | lars_rejected@snarkybark.se         |       |
+      | nils_member@bowwowwow.se            |       |
+      | admin@shf.com                       | true  |
+      | emma_under_review@happymutts.se     |       |
+      | hans_ready_for_review@happymutts.se |       |
 
-    And the following applications exist:
-      | first_name   | user_email                  | company_number | state                 |
-      | Emma         | emma@random.com             | 5562252998     | waiting_for_applicant |
-      | Hans         | hans@random.com             | 5560360793     | waiting_for_applicant |
-      | Anna         | anna_needs_info@random.com  | 2120000142     | waiting_for_applicant |
-      | LarsRejected | lars_rejected@snarkybark.se | 0000000000     | rejected              |
-      | NilsApproved | nils_member@bowwowwow.se    | 0000000000     | accepted              |
 
     And the following business categories exist
       | name         |
       | Groomer      |
       | Psychologist |
       | Trainer      |
+      | dog grooming |
+      | dog crooning |
+      | rehab        |
+
+    And the following companies exist:
+      | name                 | company_number | email                 |
+      | No More Snarky Barky | 5560360793     | snarky@snarkybarky.se |
+
+    And the following applications exist:
+      | first_name         | user_email                          | company_number | state                 | category_name |
+      | Emma               | emma@random.com                     | 5562252998     | waiting_for_applicant | Psychologist  |
+      | Hans               | hans@random.com                     | 5560360793     | waiting_for_applicant | Psychologist  |
+      | Anna               | anna_needs_info@random.com          | 2120000142     | waiting_for_applicant | Psychologist  |
+      | LarsRejected       | lars_rejected@snarkybark.se         | 0000000000     | rejected              | dog crooning  |
+      | NilsApproved       | nils_member@bowwowwow.se            | 0000000000     | accepted              | Groomer       |
+      | EmmaUnderReview    | emma_under_review@happymutts.se     | 5562252998     | under_review          | rehab         |
+      | HansReadyForReview | hans_ready_for_review@happymutts.se | 5562252998     | ready_for_review      | dog grooming  |
+
 
+  @admin
```

Comment:
I'm just using them as `tags`, not as `hooks` that call any code.

I've put those in so that I can run only certain tests:  all of the tests that an admin would do, all of the tests that user would do, etc:

`cucumber --tags @admin `